### PR TITLE
Upgrade ember/ember-cli to 2.2.0 and Deps

### PR DESCRIPTION
**Pull request by kscherling**
- Upgrades ember to 2.2.0
- Upgrade ember-cli to 2.2.0-beta.3
- Upgrade all bower/npm dependencies to match edge releases for both
- Fix deprications
- *note - there are still three deprication warnings coming from addons
  that need to be updated by author

Closes #1564

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -7,23 +7,23 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    {{content-for 'head'}}
+    {{content-for "head"}}
```
Comment (zulaica) [inline app/index.html]: I thought we were going single quotes across the board except for string interpolation?

**Comment by zulaica** [comment]
Running `bower install` gives me this prompt:

```
Unable to find a suitable version for qunit-notifications, please choose one:
    1) qunit-notifications#~0.0.6 which resolved to 0.0.6 and is required by ember-qunit-notifications#0.0.7
    2) qunit-notifications#~0.1.0 which resolved to 0.1.0 and is required by ember-qunit-notifications#0.1.0
```

**Comment by zulaica** [comment]
QUnit is broken.

screen shot 2016-01-11 at 5 30 07 pm

**Comment by zulaica** [comment]
This branch isn't pulling my local projects. Switching to master works pulls up my local projects. Side nav also isn't loading.

screen shot 2016-01-11 at 6 00 03 pm

File: app/index.html (hunk 1 above)
Comment (kscherling) [inline app/index.html]: Ya, I thought that too.  As I was going through the ember-cli diff though they are changing to double quotes in several places so want to look into further....

**Comment by kscherling** [comment]
The projects nav was acting up for me on other branches but may just have something to do with mirage... I check though

**Comment by kscherling** [comment]
Will check on quit as well

**Comment by kscherling** [comment]
Sidebar navigation links weren't rendering in other branch as well but will verify, I can't remember how far we had gotten that

**Comment by kscherling** [comment]
Okay, this is fixed and ready for review.  Some notes:

- For the Project Dropdown/Sidenav to work, certain mocks need to be commented out in `mirage/config.js` (they are commented out now so fresh pull will render site)
- To get tests to run properly, you need to uncomment the 'projects' and 'mes' MIrage endpoints in `mirage/config.js`

**Pull request merged**